src/app/workflow-editor/object-inspector/io-inspector/workflow-io-inspector.component.ts

```diff
@@ -324,16 +328,15 @@ export class WorkflowIOInspectorComponent extends DirectiveBase implements OnIni
     }
 
     labelUpdate(ev: FocusEvent) {
-        let val = (<HTMLInputElement>ev.srcElement).value;
+        const el = <HTMLInputElement>ev.srcElement;
+        let val = el.value;
```

Comment:
There is no reason to break this up into two declarations, `el` isn't used anywhere else in the scope. Also if a value isn't going to be changed, as is the case with `val`, use `const` instead of `let`.